packages/bpk-component-card-list/src/common-types.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Dispatch, ReactElement, SetStateAction } from 'react';
+
+export type layoutDesktopProps = 'row' | 'grid';
+export type layoutMobileProps = 'rail' | 'stack';
```

Comment:
For the layout types, I would create a separate enum and then have it in there as we can then export that and make it easier to use and expand if ever required

```
export const LAYOUTS = {
   row: 'row',
   grid: 'grid',
   rail: 'rail',
   stack: 'stack'
} as const;

```